### DISCUSSION TITLE
Is there anyway to configure the background of context bar

### DISCUSSION THREAD

**Discussion by Sansui233** [adoyle-h/one.nvim#11] [Q&A] [answered]
I have no idea where the context bar comes from. The outter background seems to be lualine's section, and what about the context area? Aerial seems no config here.
<img width="300" alt="Screen Shot 2023-02-05 at 05 20 49" src="https://user-images.githubusercontent.com/27474981/216790095-3e3bf8c4-1b53-4aa9-9f4e-ad9a711e6250.png">

**Comment by adoyle-h** [comment, marked as answer]
It's [a Lualine's section](https://github.com/adoyle-h/one.nvim/blob/70e3049589d66a5ace87a3d6dc00e28cbd70ea15/lua/one/plugins/lualine.lua#L287), and the code context is from [Aerial](https://github.com/stevearc/aerial.nvim#lualine).
The status bar at the top of window is the Winbar which customized by Lualine.

## To change the background of context bar

```lua
require('one').setup {
	config = {
		statusline = {
			lualine = {
				winbar = {
					lualine_c = {
						{
							'aerial',
							sep = ' > ', -- The separator to be used to separate symbols in status line.
							depth = nil,
							dense = false,
							dense_sep = '.', -- The separator to be used to separate symbols in dense mode.
							colored = true, -- Color the symbol icons.
							color = { fg = '#9ac3de', bg = '#0A1921', gui = 'bold' }, -- the color setting of context bar
						},
					},
				},
			},
		},
	}
}
```

## To hide the context bar

```lua
require('one').setup {
	config = {
		statusline = { lualine = { winbar = { lualine_c = {} } } },
	}
}
```

It will hide the context, but a blank bar is still visible. Below config will hide it completely.

```lua
require('one').setup {
	config = {
		statusline = { lualine = { winbar = { lualine_c = {}, lualine_x = {} } } },
	}
}
```

**Reply by Sansui233** [reply]
Thank you~
